tests/components/roborock/test_vacuum.py

```diff
@@ -154,3 +155,17 @@ async def test_failed_user_command(
             data,
             blocking=True,
         )
+
+
+async def test_get_maps(
+    hass: HomeAssistant, bypass_api_fixture, setup_entry: MockConfigEntry
+):
+    """Test that the service for maps correctly outputs rooms with the right name."""
+    response = await hass.services.async_call(
+        DOMAIN,
+        GET_MAPS_SERVICE_NAME,
+        {ATTR_ENTITY_ID: ENTITY_ID},
+        blocking=True,
+        return_response=True,
+    )
+    assert response[ENTITY_ID]["maps"][0]["rooms"][17] == "Example room 2"
```

Comment:
Maybe we can assert the whole response with a snapshot so it's easier to see what this service would actually output?

Reply:
Good call! I have made that change.